tests/kuttl/common/errors_cleanup_keystone.yaml

```diff
@@ -0,0 +1,121 @@
+#
+# Check for:
+#
+# No KeystoneAPI CR
+# No Deployment for KeystoneAPI CR
+# No Pods in keystone Deployment
+# No Keystone Services
+# No Keystone Routes
+#
+apiVersion: keystone.openstack.org/v1beta1
+kind: KeystoneAPI
+metadata:
+  finalizers:
+  - KeystoneAPI
+  name: keystone
+  namespace: openstack
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: keystone
+  namespace: openstack
+---
+# the openshift annotations can't be checked through the deployment above
+apiVersion: v1
+kind: Pod
+metadata:
+  annotations:
+    openshift.io/scc: anyuid
+  labels:
+    service: keystone
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    admin: "true"
+    service: keystone
+  name: keystone-admin
+  namespace: openstack
+spec:
```

Comment:
Here and below: If we only want to assert that the Service is not exists then can we simply leave out the spec part? The metadata part should identify the service unambiguously.

Reply:
Removed.